docs/voice/satellites/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Assist satellites"
+---
+
+The [Assist satellite](https://www.home-assistant.io/integrations/assist_satellite) integration represents a remote satellite that uses [pipelines](/docs/voice/pipelines). Integrations must implement the [base entity](#base-entity) for the `assist_satellite` platform. These satellites can then be controlled remotely via [actions](https://next.home-assistant.io/integrations/assist_satellite#actions) and a [WebSocket API](#websocket-api).
+
+## Base entity
```

Comment:
The entity should be documented in its own page under the entity pages:

https://developers.home-assistant.io/docs/core/entity